speakers/timothy-john-weaving.md

```diff
@@ -1,4 +1,3 @@
 name: Timothy John Weaving
 ---
-I am at present a Mathematics student at Cardiff University, going into my second year. After discovering an interest in the application of Number Theory to such fields as digital security, I wrote a short paper on Euler’s Theorem and RSA Public Key Cryptography, for which I won a ticket to PyCon 2016.
-As part of my research into the maths of RSA I explored the behaviour of Euler's "totient function" and was aided greatly by python in explaining the striking patterns which arose.
\ No newline at end of file
+I am at present a Mathematics student at Cardiff University, going into my second year. Having looked at Python in no real depth prior to the course it was not until the first semester that I had a proper introduction to the language. As part of my research into the maths of RSA I explored the behaviour of Euler's "totient function" and was aided greatly by python in explaining the striking patterns which arose.
```

Comment:
@alexwlchan You're quite right, but we haven't had the time to go through and check all the speakers' abstracts and biographies for spelling and grammar.  I can fix this one easily though.